scheduler/src/cook/rest/api.clj

```diff
@@ -3398,6 +3515,40 @@
                           307 {:description "Redirecting request to leader node."}
                           400 {:description "Invalid request format."}}}}))
 
+        (c-api/context
+          "/compute-clusters" []
+          (c-api/resource
+            {:produces ["application/json"]
+
+             :post
+             {:summary "Create a new compute cluster"
+              :parameters {:body-params InsertComputeClusterRequest}
+              :handler (post-compute-clusters-handler conn
+                                                      is-authorized-fn
+                                                      leadership-atom
+                                                      leader-selector)
+              :responses {201 {:description "The compute cluster was created."}
+                          307 {:description "Redirecting request to leader node."}
+                          409 {:description "There is already a compute cluster with the given name."}}}
```

Comment:
This text seems wrong, as we look for and do allow patching an existing one. Maybe :patch HTTP verb, or updat  this description?